README.md

```diff
@@ -4,37 +4,64 @@
+The first player to call for a match of Jollyballs shall be the first captain. The player who seconds the notion shall be the second captain. Selecting players will then occur in the form of a Snake draft. The first captain drafts a single player, the second drafts two players, and then the remaining undrafted player joins the first captain's team.
 
 ## Choosing which team serves first:
 Before each game in the series, each team places their paddles on the table with their paddle handle hanging off the edge. Two paddles should be placed on the corner and the third paddle should be centered on the table.
 
 One team will toss the jollyball at the other team's paddles. If the first player hits the padding of the paddle, then the other team has a chance for a rebuttal. In the event that the player of the second team hits the paddle or handle, the second team will be first to serve. If the first player hits the handle of any of the second team's paddles, or hits two paddles in one shot, then it will be their ball first.
 
+In the event that every player on each team has had a chance to shoot, the rebuttal is suspended and the first team to hit a paddle wins.
+
 *Note: If a player's elbow extends past the edge of the table, they forfeit their toss, giving the ball to the other side.*
 
 ## How to Play:
 The team who won the initial toss will start with the ball. The person in the center of the table will act as the server and will serve as though they were playing ping pong (the ball must hit the paddle then the server's side of the table before going over the net). As is the case with ping pong, a let is grounds for a re-serve.
 
 The receiving team must let the ball bounce on their side before passing the ball to a teammate. When the ball crosses the net, it cannot be immediately returned and must be hit by at least two players before being returned. The ball can be returned after two or three touches, but cannot exceed three or it will result in the loss of the ball/point.
 
-In the event of a slam, the ball can be blocked or deflected back to the side that slammed without the need for a pass. A block does not count against the three maximum touches, nor does it count against the team that slammed. However, if a block is hit back to the team who slammed the ball, that team has to return the ball, as the ball is live.
+If a player is hit while outside the field of play, the ball is dead unless it hits the following locations; the edge of the table, a player's paddle, a player's wrist or hand, or any other body part over the table.
+
+##Blocks
+Any shot, slam, poke, or hit can be blocked, with the exception of a serve. A shot can be blocked or deflected back to the side that hit the ball without the need for a pass and will count as a live ball. A block does not count against the three maximum touches and a player may allow immediately hit the ball after blocking. nor does it count against the team that hit it.
```

Comment:
I think you can get rid of that whole last sentence: `. nor does it count against the team that hit it.`